mozregression/main.py

```diff
@@ -240,6 +251,10 @@ def cli(argv=None):
         test_runner = CommandTestRunner(fetch_config, options.command,
                                         persist=options.persist)
 
+    prefs = mozprofile.prefs.Preferences()
+    if options.preferences:
+        prefs.add_file(options.preferences)
+
```

Comment:
Sorry for the delay. I wanted to confirm that from the above link you want me to implement something similar to the preferences function, right? and secondly the --pref argument is to be implemented in a similar way as the CLI preferences have been implemented, right?